zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/SimpleTensorFlowPredictorBuilder.java

```diff
@@ -0,0 +1,134 @@
+/*-
+ * -\-\-
+ * zoltar-tensorflow
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.zoltar.tf;
+
+import com.spotify.featran.FeatureSpec;
+import com.spotify.futures.CompletableFutures;
+import com.spotify.zoltar.FeatureExtractFns;
+import com.spotify.zoltar.ModelLoader;
+import com.spotify.zoltar.Prediction;
+import com.spotify.zoltar.Predictor;
+import com.spotify.zoltar.featran.FeatranExtractFns;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import org.tensorflow.Session;
+import org.tensorflow.Tensor;
+import org.tensorflow.Tensors;
+import org.tensorflow.example.Example;
+
+/**
+ * Helps to build a simple TensorFlow predictor.
+ * In this context, a simple TensorFlow predictor has the following characteristics:
+ * - Has a single input op
+ * - Has a single output op
+ * - Uses Featran to extract and consume Example objects
+ * The output of the Predictor will always be a {@link JTensor}.
+ */
+public class SimpleTensorFlowPredictorBuilder<InputT> {
```

Comment:
@yonromai can we make this an `@AutoValue` class with an `@AutoValue.Builder`?

Reply:
@yonromai I think we are moving in the direction that `PredictorBuilder`'s are sort of value-typed objects that can be decorated/composed, allowing you to get a `Predictor` based on that context. `master` reflects this now https://github.com/spotify/zoltar/blob/master/zoltar-core/src/main/java/com/spotify/zoltar/DefaultPredictorBuilder.java#L38